src/main/java/com/sap/oss/phosphor/fosstars/tool/format/CommonFormatter.java

```diff
@@ -384,14 +393,4 @@ static String loadFrom(String resource, Class<?> clazz) {
       throw new UncheckedIOException("Holy moly! Could not load template!", e);
     }
   }
-
-  /**
-   * Formats a double.
-   *
-   * @param n The double to be formatted.
-   * @return A formatted string.
-   */
-  static String formatted(double n) {
-    return DECIMAL_FORMAT.format(n);
-  }
 }
```

Comment:
We need to add the feature to be displayed as part of the report in the final output, We need to add a line in the top of this file static block like `add(HAS_SECURITY_POLICY, "Does it have a security policy?");`